app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None | ValueError:
+    if command.split()[0] != "cp" or len(command.split()) != 3:
+        raise ValueError("Invalid command format")
+
+    original, copy = command.split()[1], command.split()[2]
```

Comment:
you can unpack these variables in the easiest way, also use the more descriptive names
command, old_file, new_file = command.split()